### ISSUE TITLE
Add option to ignore mapping errors

### ISSUE THREAD

**Issue by aaberg** [aaberg/sql2o#134]
Ref discussion in issue #132 

@dimzon I have assigned this to you. Is that OK?

**Comment by natinusala** [comment]
Any ETA for this feature ?

**Comment by aaberg** [comment]
It is one of the most requested features now! I have changed the priority to high, and will try to implement it as fast as possible, preferably within a week

**Comment by aaberg** [comment]
This is now fixed. Will be released with version 1.5.4

The syntax is:

```java
connection.createQuery(sql)
    .throwOnMappingFailure(false)
    .executeAndFetchFirst(Pojo.class);
```

**Comment by natinusala** [comment]
Thanks a lot :)
